stemming/stemming.py

```diff
@@ -1,7 +1,7 @@
 import copy
 import time
 import logging
-import pickle5 as pickle
+import pickle
 import requests
```

Comment:
this suppose to be stashed (it served you local env right? we dont want it to go to prod server)